packages/aws-cdk/test/integ/cli/cdk-helpers.ts

```diff
@@ -39,6 +39,7 @@ export interface ShellOptions extends child_process.SpawnOptions {
 
 export interface CdkCliOptions extends ShellOptions {
   options?: string[];
+  neverRequireApproval?: boolean;
```

Comment:
I prefer if we could avoid the negation in the name (i.e. prefer requireApproval). 
In this case its closely tied to the flag we want to set so I don't mind it